src/js/AppHeader.js

```diff
@@ -64,7 +64,11 @@ class AppHeader extends React.Component {
         var icon = this.props.icon == 'false' ? (<div />) : (<MenuIcon isShowingMenu={isShowingMenu}/>);
 
         if (this.props.icon == 'store') {
-            icon = this.props.router.isActive('/appstore') ? (<AppStoreMenuIcon />) : (<AppStoreIcon />);
+            if(this.props.isAppStoreConnected){
```

Comment:
```suggestion
            if (this.props.isAppStoreConnected) {
```
you dont like spaces? i think it would be nice to keep the style consistent

Reply:
lol I wouldn't go that far, just not in the habit of adding them consistently. This one and a few others fixed in https://github.com/smartdevicelink/generic_hmi/pull/241/commits/a6eedda259a03dff1fed10ce568e10a4aa497436